src/traces/pie/plot.js

```diff
@@ -170,6 +170,11 @@ module.exports = function plot(gd, cdpie) {
                 }
 
                 function handleClick() {
+                    // TODO: this does not support right-click. If we want to support it, we
+                    // would likely need to change mapbox to use dragElement instead of straight
```

Comment:
we would need to change _pie_ to use dragElement,

correct?

Reply:
Perhaps we could discuss future `dragElement` generalizations in https://github.com/plotly/plotly.js/issues/2057? 

`table`, `sankey` and `parcoords` would also benefit from using `dragElement`.

Reply:
comment added to #2057 - I'll leave the TODOs present in the code then unless you object.